lnrpc/rpc.proto

```diff
@@ -891,6 +891,13 @@ message EstimateFeeRequest {
     // The target number of blocks that this transaction should be confirmed
     // by.
     int32 target_conf = 2;
+
+    // The minimum number of confirmations each one of your outputs used for
+    // the transaction must satisfy.
+    int32 min_confs = 3;
```

Comment:
Please split the changes in the main RPC (`rpc.proto` and `rpcserver.go`) into a separate commit and the changes in the `walletrpc` package into another.